.gitignore

```diff
@@ -378,6 +378,9 @@ ENV/
 env.bak/
 venv.bak/
 
+#conda-build
+conda_bld/
+
```

Comment:
Do we want this here? I don't even call my build folders this

Reply:
I actually would also leave the first one! It's `.gitignore`, I'm fine ignoring anything people find useful.